### DISCUSSION TITLE
How to use it without a dedicated express process but in SvelteKit?

### DISCUSSION THREAD

**Discussion by lkj4** [RobinTail/express-zod-api#601] [Q&A] [answered]
Pretty much the title. I wan to save myself to to run another express process and rather let express-zod-api run within SvelteKit, e.g. launched in the hook.ts file

Is this possible?

**Comment by RobinTail** [comment, marked as answer]
that might be `sveltekit-zod-api` :)
at least per my current understanding of how the mentioned SvelteKit works.
This library meant to be a "kit" as well, by combining a webserver, validator, router, documentation and client generator.
I'm not sure how it could be possible to combine it with SvelteKit.
But in case you have any ideas on that, I'd be glad to discuss 'em.
